From eaa64b83f69d095654702de9e0a37ef57bc26a35 Mon Sep 17 00:00:00 2001
From: Simon Horman <horms+renesas@verge.net.au>
Date: Wed, 3 Jul 2013 13:38:25 +0900
Subject: [PATCH 0955/1083] Revert "drm/rcar-du: Name the encoder platform data
 union"

This reverts commit 441bea3f7e1abd0000fc5523a565c851633e0427.
(cherry picked from commit 577d95048f6d0232852316e2fdbb38ff81156377)

Signed-off-by: Ryo Kataoka <ryo.kataoka.wt@renesas.com>
---
 drivers/gpu/drm/rcar-du/rcar_du_kms.c | 4 ++--
 include/linux/platform_data/rcar-du.h | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/gpu/drm/rcar-du/rcar_du_kms.c b/drivers/gpu/drm/rcar-du/rcar_du_kms.c
index 533aa97..d609e9d 100644
--- a/drivers/gpu/drm/rcar-du/rcar_du_kms.c
+++ b/drivers/gpu/drm/rcar-du/rcar_du_kms.c
@@ -164,11 +164,11 @@ int rcar_du_modeset_init(struct rcar_du_device *rcdu)
 
 		switch (pdata->encoder) {
 		case RCAR_DU_ENCODER_VGA:
-			rcar_du_vga_init(rcdu, &pdata->u.vga, i);
+			rcar_du_vga_init(rcdu, &pdata->vga, i);
 			break;
 
 		case RCAR_DU_ENCODER_LVDS:
-			rcar_du_lvds_init(rcdu, &pdata->u.lvds, i);
+			rcar_du_lvds_init(rcdu, &pdata->lvds, i);
 			break;
 
 		default:
diff --git a/include/linux/platform_data/rcar-du.h b/include/linux/platform_data/rcar-du.h
index c315d32..0f2c836 100644
--- a/include/linux/platform_data/rcar-du.h
+++ b/include/linux/platform_data/rcar-du.h
@@ -41,7 +41,7 @@ struct rcar_du_encoder_data {
 	union {
 		struct rcar_du_encoder_lvds_data lvds;
 		struct rcar_du_encoder_vga_data vga;
-	} u;
+	};
 };
 
 struct rcar_du_platform_data {
-- 
1.8.3.2

